common/presenters/moves-to-table.test.js

```diff
@@ -148,13 +148,13 @@ describe('#movesToTable', function() {
           html: 'Name',
         },
         {
-          html: 'Created at',
+          html: 'Sent on',
```

Comment:
@barbaracassani These tests failing when I updated the content in the locales file worried me a bit.

Because this component is now using the other two, I don't think the i18n mocks are really working.

I'm wondering if its cleaner to just have separate transformers for moves/allocations -> table component rather than trying to share the head/rows objects. It feels like its not DRYing it up that much and making it a bit trickier to test. As if we want to mock the deps in this test we have to use proxyquire which always feels a bit annoying. What are your thoughts?

Reply:
Addressed in my own pr #476